Formula/docuum.rb

```diff
@@ -18,6 +18,16 @@ def install
     system "cargo", "install", *std_cargo_args
   end
 
+  plist_options manual: "docuum"
+  # https://github.com/stepchowfun/docuum#configuring-your-operating-system-to-run-the-binary-as-a-daemon
+  service do
+    run [opt_bin/"docuum", "--threshold", "15 GB"]
```

Comment:
Just wondering, does the user have the ability to customize this threshold somehow?

Reply:
Not that I know of, unfortunately.

Reply:
In that case, perhaps it is better to not hardcode a threshold here and just rely on Docuum's default ([currently](https://github.com/stepchowfun/docuum/blob/5959ff7fab0be2fbafbfde2bed9fbdd8a7806eae/src/main.rs#L32) `10 GB`):

```ruby
run [opt_bin/"docuum"]
```

What do you think?

Reply:
Works for me.